src/components/CountryBoard.vue

```diff
@@ -1,7 +1,119 @@
-<script lang="ts" setup></script>
+<script lang="ts" setup>
+import { useCountriesStore } from "@/stores/countries.store";
+import { computed, onMounted } from "vue";
+import { ICountry } from "@/types/countries";
+
+const store = useCountriesStore();
+const isLoading = computed((): boolean => store.$state.isLoading);
+const countriesList = computed((): ICountry[] => store.$state.countriesList);
+const countryBasicInfoTitles = [
+  {
+    title: "Population",
+    key: "population",
+  },
+  {
+    title: "Region",
+    key: "region",
+  },
+  {
+    title: "Capital",
+    key: "capital",
+  },
+];
+
+onMounted(() => {
```

Comment:
czemu robisz to na onMounted

Reply:
no to robisz na mounted, czy created? to nie jest to samo

Reply:
z czasem jak na to patrze lepiej zrobić to na created. Bo tak najpierw wygeneruje Ci sie dom, a dopiero potem strzelasz do API, lepiej od razu niech uderza budując DOM

Reply:
skorzystam z onBeforeMount